v3/pkg/application/systemtray_windows.go

```diff
@@ -4,10 +4,11 @@ package application
 
 import (
 	"fmt"
-	"github.com/wailsapp/wails/v3/pkg/icons"
 	"syscall"
 	"unsafe"
 
+	"github.com/wailsapp/wails/v3/pkg/icons"
```

Comment:
A bit pedantic but would you move this all the way down to the rest of the local imports please?

Reply:
These get shifted around all the time due to editor preference. Let's look at that separately. Perhaps a .editor file will help, may not 😅